sql-cli/sql_cli/sql_directory_parser.py

```diff
@@ -99,15 +99,17 @@ def get_relative_target_path(self) -> str:
 
         :returns: the path where sql files without any headers are being placed.
         """
-        target_full_directory = self.target_directory / "/".join(self.get_sub_directories())
+        target_full_directory = (
+            self.target_directory / self.root_directory.name / "/".join(self.get_sub_directories())
+        )
         target_full_directory.mkdir(parents=True, exist_ok=True)
 
         target_path = target_full_directory / self.path.name
 
         with open(target_path, "w") as file:
             file.write(f"{self.content}\n")
 
-        return target_path.relative_to(self.root_directory.parent).as_posix()
+        return target_path.relative_to(self.root_directory.parent.parent).as_posix()
```

Comment:
Having `parent.parent` or `parent.parent.parent` feels like technical debt. It's ok for now but I don't want this to be how this is released 1.0.

Reply:
I agree. But I could not get it to work quickly without all these parents :D